Hero.h

```diff
@@ -55,33 +57,41 @@ class Hero : public Unit{
 	*  \param magical_damage_bonus_per_level [in] Hős mágikus sebézésének növekedése szintekként.
   	*  \param cooldown_multiplier_per_level [in] Hős támadási idő csökkenés mértéke szintekként.
 	*  \param defense_bonus_per_level [in] Hős védelmi növekedése szintekként.
+	*  \param light_radius [in] Hős látótávolsága.
+	*  \param light_radius_bonus_per_level [in] Hős látótávolsága növekedése szintekként.
```

Comment:
Előre is bocsánat, de nagyon zavarta a szemem, szintenként*

Reply:
Köszönöm szépen Tamás az észrevételt. Javítottam.